e2e/cypress/integration/websocket/channel_created/new_sidebar_spec.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @websocket
+
+import {testWithConfig} from '../../../support/hooks';
+
+import {getRandomId} from '../../../utils';
+
+describe('Handle removed user - new sidebar', () => {
+    testWithConfig({
+        ServiceSettings: {
+            ExperimentalChannelSidebarOrganization: 'default_on',
+        },
+    });
+
+    it('MM-27202 should add new channels to the sidebar when created from another session', () => {
+        // # Start with a new team
+        const teamName = `team-${getRandomId()}`;
+        cy.createNewTeam(teamName, teamName);
+
+        // * Verify that we've switched to the new team
+        cy.get('#headerTeamName').should('be.visible').should('contain', teamName);
+
+        // # Create a new channel from another session
+        const channelName = `channel-${getRandomId()}`;
+        cy.getCurrentTeamId().then((currentTeamId) => {
```

Comment:
would it be worth verifying `#sidebarItem_${channelName}` does _not_ exist before creating it? I know chances are low that the channel would already exist, but an extra assertion of it not existing wouldn't hurt.

Reply:
I don't think that should be necessary in this case since we've created a new team for each of these tests, so there won't be any channels in the team to begin with.

Reply:
Ah true good point.